modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -82,36 +83,43 @@ DEFAULT pxeboot
 TIMEOUT 20
 PROMPT 0
 LABEL pxeboot
-    KERNEL http://<HTTP_server>/rhcos-<version>-installer-live-kernel-<architecture> <1>
-    APPEND initrd=http://<HTTP_server>/rhcos-<version>-installer-live-initramfs.<architecture>.img console=ttyS0 console=tty0 coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-installer-live-rootfs.<architecture>.img <2>
+    KERNEL http://<HTTP_server>/rhcos-<version>-live-kernel-<architecture> <1>
+    APPEND initrd=http://<HTTP_server>/rhcos-<version>-live-initramfs.<architecture>.img coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-live-rootfs.<architecture>.img coreos.inst.install_dev=/dev/sda coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <2> <3>
 ----
 <1> Specify the location of the live `kernel` file that you uploaded to your HTTP
 server.
-<2> Specify locations of the {op-system} files that you uploaded to your HTTP
-server. The `initrd` parameter value is the location of the live `initramfs`
-file, the `coreos.inst.ignition_url` parameter value is the location of the
-bootstrap Ignition config file, and the `coreos.live.rootfs_url` parameter value is
-the location of the live `rootfs` file.
+The URL must be http, tftp, or ftp; https and nfs are not supported.
```

Comment:
I would capitalize each of these values, or put them in back ticks, for readability.

Reply:
I'll capitalize, but no on the back ticks. We hare not referring to them as literal values here, but rather as supported protocols.